processing/data_collection/gazette/spiders/pr_cascavel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from dateparser import parse
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCascavelSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4104808'
+    name = 'pr_cascavel'
+    allowed_domains = ['cascavel.pr.gov.br']
+    start_urls = ['http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php']
+    download_url = 'http://www.cascavel.pr.gov.br/anexos/{}'
+
+    def parse(self, response):
+        """
+        @url http://www.cascavel.pr.gov.br/servicos/orgao_oficial.php
+        @returns items 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        start_date = dt.date(2015, 1, 1)
```

Comment:
Current strategy is to allow the spider to run through all the dates and the pipeline drops gazettes earlier than the period of interest. (this allows us to change the period later in a single place see #39)